base/src/org/compiere/process/EMailTest.java

```diff
@@ -18,8 +18,7 @@
 
 import java.io.File;
 
-import org.compiere.model.MClient;
-import org.compiere.model.MStore;
+import org.compiere.model.*;

```

Comment:
@rcapecce in ADempiere the best practice indicates that you define exactly the classes that are being imported and the use of .* to import the classes is not recommended, please change it

Reply:
ok @e-Evolution , I change this imports